### PR TITLE
add my updated script -Adunola

### PR DESCRIPTION
**Pull request by adun-dotcom**

### CODE REVIEW

**Comment by Marvellous1** [comment]
Your code passes but there is no need for that last line

**Comment by Marvellous1** [comment]
But how many files do you have

File: Brown-coder.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+let fullName= "Adesuwa Adolosa";
+let id= "HNG-02362";
+let lang= "Javascript";
+let email = "adesuwahealthalive@gmail.com";
+console.log(`Hello World, this is ${fullName} with HNG ID: ${id} and email address: ${email}, using ${lang} for stage 2 task` )
```
Comment (BenjaminKyamanywa) [inline Brown-coder.js:5]: hey, @Brown-coder, edit your console log output to follow this ; 
**Hello World, this is [full name] with HNGi7 ID [ID] and email [email] using [language] for stage 2 task**
remove the full colons in your output as they are not part of the guided output ':'

File: Brown-coder.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+let fullName= "Adesuwa Adolosa";
+let id= "HNG-02362";
+let lang= "Javascript";
```
Comment (BenjaminKyamanywa) [inline Brown-coder.js:3]: use 'language' instead of 'lang' for your variable

File: Brown-coder.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+let fullName= "Adesuwa Adolosa";
```
Comment (BenjaminKyamanywa) [inline Brown-coder.js:1]: you can use 'name' instead of 'fullName' for this variable.

File: Brown-coder.js (hunk 1 above)
Comment (Brown-coder) [reply Brown-coder.js:5]: thanks

**Comment by Rubylena** [comment]
cant merge this. too many conflicts. resend a correct PR

**Pull request closed without merging**